tests/ui/test_ui_tools.py

```diff
@@ -19,6 +19,7 @@
 
 VIEWS = "zulipterminal.ui_tools.views"
 TOPBUTTON = "zulipterminal.ui_tools.buttons.TopButton"
```

Comment:
Do we need `TOPBUTTON` here any more?

Reply:
Yeah. It was used by another test somewhere (in this same file);

Reply:
Yup, the test for update count requires it. Which occurs in a later commit. I can remove it here and reintroduce it in the later one. But i though to let it be - as is; is that okay?